enhancements/observability/multicluster-observability-addon.md

```diff
@@ -0,0 +1,712 @@
+
+For the `otlp` exporter the user need to provide a ConfigMap `tracing-spoke-otlp` with all the required per-cluster fields:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tracing-spoke-otlp
+  namespace: spoke
+  labels:
+    mcoa.openshift.io/signal: tracing
+  annotations:
+    tracing.mcoa.openshift.io/target-output-name: otlp
+data:
+  url: https://mycompany.grafana.com/api/traces/v1/spoke
+```
+
+
+For the `otlp` exporter MCOA will provide a Secret `tracing-otlp-auth` with all required authentication information, e.g. TLS client Certificate:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: tracing-otlp-auth
+  namespace: spoke
+stringData:
+  'tls.crt': "TLS client certificate"
+  'tls.key': "TLS key"
+  'ca-bundle.crt': "Certificate Authority certificate" # Pull from the referenced Secret otel-gateway
+```
+
+In turn the addon will compile a `ManifestWork` for the managed cluster `spoke` as follows and pass it over it's WorkAgentController:
+
+```yaml
+kind: ManifestWork
+metadata:
+  name: addon-multicluster-observability-addon-deploy-0
+  namespace: spoke
+spec:
+  workload:
+    manifests:
+    - apiVersion: project.openshift.io/v1
+      kind: Project
+      metadata:
+        name: openshift-opentelemetry-operator
+    - apiVersion: operators.coreos.com/v1
+      kind: OperatorGroup
+      metadata:
+        name: openshift-opentelemetry-operator-ghl6v
+        namespace: openshift-opentelemetry-operator
+      spec:
+        upgradeStrategy: Default
+    - apiVersion: operators.coreos.com/v1alpha1
+      kind: Subscription
+      metadata:
+        name: opentelemetry-product
+        namespace: openshift-opentelemetry-operator
+      spec:
+        channel: stable
+        installPlanApproval: Automatic
+        name: opentelemetry-product
+        source: redhat-operators
+        sourceNamespace: openshift-marketplace
+        startingCSV: opentelemetry-operator.v0.81.1-5
+    - apiVersion: v1
+      kind: Secret
+      metadata:
+        name: tracing-otlp-auth
+        namespace: spoke
+      stringData:
+        'tls.crt': "TLS client certificate"
+        'tls.key': "TLS key"
+        'ca-bundle.crt': "Certificate Authority certificate"
+    - apiVersion: opentelemetry.io/v1alpha1
+      kind: OpenTelemetryCollector
+      metadata:
+        name: spoke-otelcol
+        namespace: open-cluster-management
+      spec:
+        config: |
+          receivers:
+            jaeger:
+              protocols:
+                grpc:
+            otlp:
+              protocols:
+                grpc:
+                http:
+          processors:
+          exporters:
+            debug:
+            otlp:
+              tls:
+                insecure: true
+          service:
+            pipelines:
+              traces:
+                receivers: [jaeger,otlp]
+                processors: []
+                exporters: [otlp, debug]
+```
+
+
+#### Hypershift [optional]
+
+TBD
+
+### Drawbacks
+
+TBD
```

Comment:
Can we describe very short what's the drawback of not having a new custom resource or extending the MultiClusterObservability resource? Also why we are not doing this at this stage?